src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+                      id="zipcode"
+                      label={`*${_('checkoutProcessForm.contact_information_zip_code')}`}
+                      withNameValidation
+                      required
+                      className="field-item--50"
+                    />
+                    <PhoneFieldItem
+                      fieldName={fieldNames.phone}
+                      label={`*${_('checkoutProcessForm.contact_information_phone')}`}
+                      placeholder={_('forms.placeholder_phone')}
+                      className="field-item--50"
+                      required
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+              <FieldItem className="field-item">
+                <fieldset>
+                  <FieldGroup>
+                    <InputFieldItem
+                      type="text"
+                      fieldName={fieldNames.company}
+                      id="company"
+                      label={`${_('checkoutProcessForm.contact_information_company')}`}
+                      className="field-item--50"
+                    />
+                    <InputFieldItem
+                      type="text"
+                      label={`${_('checkoutProcessForm.contact_information_industry')}`}
+                      fieldName={fieldNames.industry}
+                      id="industry"
+                      className="field-item--50"
+                    />
+                  </FieldGroup>
+                </fieldset>
+              </FieldItem>
+            </FieldGroup>
+            {formSubmitted ? (
+              <div className="dp-wrap-confirmation" style={containerMessageSucces}>
+                <div className="dp-msj-confirmation bounceIn" style={flextStartSyle}>
+                  <p>{_('checkoutProcessForm.success_msg')}</p>
+                </div>
+              </div>
+            ) : null}
+            <div className="dp-footer-form" style={flextStartSyle}>
+              <div className="dp-action">
+                <SubmitButton className="dp-button button-medium primary-green">
+                  {_('checkoutProcessForm.save_continue')}
+                </SubmitButton>
+              </div>
+            </div>
+          </fieldset>
+        </FormWithCaptcha>
+      ) : null}
+    </div>
+  );
+};
+
+const CheckoutProcess = ({
+  location,
+  dependencies: { planService, dopplerUserApiClient, appSessionRef },
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const { pathType, planType } = useRouteMatch().params;
+
+  const [steps, setSteps] = useState({
+    contactInformation: true,
+    billingInformation: false,
+    paymentInformation: false,
+  });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep();
```

Comment:
Here you could have something like this:
```suggestion
      setStep('contact-information');
```